conf/azurebatch.config

```diff
@@ -1,25 +1,64 @@
 //Nextflow config file for running on Azure batch
 params {
   config_profile_description = 'Azure BATCH Cloud Profile'
-  config_profile_contact = 'Venkat Malladi (@vsmalladi)'
+  config_profile_contact = 'Venkat Malladi (@vsmalladi) & Adam Talbot (@adamrtalbot)'
   config_profile_url = 'https://azure.microsoft.com/services/batch/'
 
-  az_location = 'westus2'
-  batch_name = false
-  batch_key = false
-  storage_name = false
-  sas_token = false
-}
+  // Storage
+  storage_name      = null
+  storage_key       = null
+  storage_sas       = null
 
-docker {
-  enabled = true
-}
```

Comment:
no need for that?

Reply:
Shouldn't I think it was added by mistake previously

Reply:
Docker is used by default

Reply:
Agreed. Don't think that is necessary now.